setuptools/config.py

```diff
@@ -262,22 +261,24 @@ def _parse_file(cls, value):
 
         spec = value[len(include_directive):]
         filepaths = (os.path.abspath(path.strip()) for path in spec.split(','))
+        return '\n'.join(
+            self._read_local_file(path)
+            for path in filepath
+            if os.path.isfile(path)
+        )
```

Comment:
@jaraco 

You changed the behavior here. If no file was read, the method returned original `value`, not an empty string.

This would correct the flow:
```python
return (...) or value
```

Reply:
Hmm. I'll have to look more closely.

Reply:
Although, if the tests aren't capturing that use-case, then it's not supported.